comms/Account.cpp

```diff
@@ -85,15 +87,39 @@ COMMS_API bool VerifyUser(IConfig * config, const CString &user, const CString &
                     retMsg += response.Exceptions->Exception[i]->Message->c_str();
                 }
             }
+        } 
+        else {
+            _ns1__MyAccountRequest request;
+            _ns1__MyAccountResponse response;
+            ESP_EXCEPTION_LOG3(response.Exceptions);
+            if (server.MyAccount(&request, &response) == SOAP_OK)
+            {
+                SAFE_ASSIGN(retPasswordExpirationWarningDays, response.passwordExpirationWarningDays);
+                SAFE_ASSIGN(retPasswordDaysRemaining, response.passwordDaysRemaining);
+
+                CString responseMessage;
+                if (responseMessage.IsEmpty() && response.Exceptions)
+                {
+                    for (unsigned int i = 0; i < response.Exceptions->Exception.size(); ++i)
+                    {
+                        if (response.Exceptions->Exception[i]->Message)
+                        {
+                            if (!retMsg.IsEmpty())
+                                retMsg += _T("\r\n");
+                            retMsg += response.Exceptions->Exception[i]->Message->c_str();
+                        }
+                    }
+                }
+                return true;
+            }
```

Comment:
Move this code into its own method and call it from here.

Reply:
@dehilsterlexis bump

Reply:
@GordonSmith done

Reply:
@GordonSmith will do